spec/lib/integrations/oauth_spec.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
+require 'rails_helper'
```

Comment:
To run the spec by itself, yeah. It seems like it doesn't auto-load without rails_helper.